app/modules/asset_groups/schemas.py

```diff
@@ -42,6 +54,57 @@ class Meta(BaseAssetGroupSightingSchema.Meta):
         dump_only = BaseAssetGroupSightingSchema.Meta.dump_only
 
 
+class AssetGroupSightingAsSightingSchema(BaseAssetGroupSightingSchema):
+    """
+    In order for the frontend to render an AGS with the same code that renders a
+    sighting, we have to pop out all the fields in AGS.config into the top-level
+    of the schema. This is done with AssetGroupSighting.config_field_getter,
+    which creates a getter for a config field of a given name. We can add more
+    fields using the pattern below.
+    """
+
+    assets = base_fields.Function(AssetGroupSighting.get_assets)
+    completion = base_fields.Function(AssetGroupSighting.get_completion)
+
+    # Note: these config_field_getter vars should conform to SIGHTING_FIELDS_IN_AGS_CONFIG
+    # at the top of this file
+    decimalLatitude = base_fields.Function(
+        AssetGroupSighting.config_field_getter('decimalLatitude')
+    )
+    decimalLongitude = base_fields.Function(
+        AssetGroupSighting.config_field_getter('decimalLongitude')
+    )
+    locationId = base_fields.Function(
+        AssetGroupSighting.config_field_getter('locationId')
+    )
+    startTime = base_fields.Function(AssetGroupSighting.config_field_getter('startTime'))
+    encounters = base_fields.Function(
+        AssetGroupSighting.config_field_getter('encounters')
+    )
+
+    class Meta(BaseAssetGroupSightingSchema.Meta):
+        # adds 'stage' to the fields already defined above
+        fields = BaseAssetGroupSightingSchema.Meta.fields + (
```

Comment:
Don't go back to listing the fields, you just need to not inherit from `BaseAssetGroupSightingSchema.Meta`.  To fix this:

```diff
diff --git a/app/modules/asset_groups/schemas.py b/app/modules/asset_groups/schemas.py
index 7d954ec4..208284cb 100644
--- a/app/modules/asset_groups/schemas.py
+++ b/app/modules/asset_groups/schemas.py
@@ -82,18 +82,9 @@ class AssetGroupSightingAsSightingSchema(BaseAssetGroupSightingSchema):
         AssetGroupSighting.config_field_getter('encounters')
     )
 
-    class Meta(BaseAssetGroupSightingSchema.Meta):
+    class Meta:
         # adds 'stage' to the fields already defined above
-        fields = BaseAssetGroupSightingSchema.Meta.fields + (
-            AssetGroupSighting.stage.key,
-            'assets',
-            'completion',
-            'decimalLatitude',
-            'decimalLongitude',
-            'locationId',
-            'startTime',
-            'encounters',
-        )
+        additional = ('stage',)
         dump_only = BaseAssetGroupSightingSchema.Meta.dump_only
 
     # Ensures we don't return fields with None values
```

Reply:
Thanks, done.